fontir/src/serde.rs

```diff
@@ -110,6 +117,12 @@ impl From<MiscSerdeRepr> for MiscMetadata {
             vendor_id: from.vendor_id,
             underline_thickness: from.underline_thickness.into(),
             underline_position: from.underline_position.into(),
+            version_major: from.version_major,
+            version_minor: from.version_minor,
+            head_flags: from.head_flags,
+            lowest_rec_ppm: from.lowest_rec_ppm,
+            created: from.created,
+            modified: from.created,
```

Comment:
hm `modified: from.created` feels suspicious..
Do we actually need to store `modified` in the StaticMetadata IR? None of the source formats allow to override this currently (only head.created), the only way for user to _not_ set it to the current time is to export SOURCE_DATE_EPOCH right now.

Reply:
Oh good call, will remove modified.

Reproducible builds would require modified *not* get updated every cycle but that can wait.

Reply:
> Reproducible builds would require modified not get updated every cycle but that can wait.

I thought we did support SOURCE_DATE_EPOCH for repro builds, don't we?

Reply:
Fixed.